helix-core/src/syntax.rs

```diff
@@ -85,17 +85,17 @@ pub struct LanguageConfiguration {
     // first_line_regex
     //
     #[serde(skip)]
-    pub(crate) highlight_config: OnceCell<Option<Arc<HighlightConfiguration>>>,
+    pub highlight_config: OnceCell<Option<Arc<HighlightConfiguration>>>,
```

Comment:
Were these made `pub` because of usage in tests? Why not use the associated methods `highlight_config()` and `indent_query()`?

Reply:
Ah, it's because `LanguageConfiguration` is constructed directly in the test. Let's instead use the loader with a custom `languages.toml`? Can reuse this one: https://github.com/helix-editor/helix/blob/ce8311727f32975dc1c1030c676ffc141f63b851/.github/workflows/languages.toml from #1659